src/components/Widget/Widget.style.ts

```diff
@@ -0,0 +1,47 @@
+import styled from "styled-components";
+import fonts from "../../styles/fonts";
+import colors from "../../styles/colors";
+import resetCSS from "../../styles/reset";
+
+export const HintStyled = styled.div`
+  position: absolute;
+  top: 16px;
+  right: 5px;
+`
+
+export const WidgetStyled =  styled.div`
+  ${resetCSS}
+  ${fonts.text};
+  align-items: center;
+  border: 1px solid ${colors.paleBlue2};
+  border-radius: 12px;
+  box-shadow: inset 0 0 10px rgba(46, 125, 175, 0.15);
+  display: flex;
+  gap: 5px;
+  padding: 16px;
+  position: relative;
+  width: 100%;
+
```

Comment:
remove this line please, is the formatting still not working on PC?

Reply:
Bill Gates does not want me to write pretty code :(